sfge/src/main/java/com/salesforce/graph/build/StaticBlockUtil.java

```diff
@@ -0,0 +1,345 @@
+package com.salesforce.graph.build;
+
+import com.salesforce.apex.jorje.ASTConstants;
+import com.salesforce.apex.jorje.JorjeNode;
+import com.salesforce.collections.CollectionUtil;
+import com.salesforce.exception.ProgrammingException;
+import com.salesforce.graph.Schema;
+import com.salesforce.graph.ops.MethodUtil;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+
+/**
+ * Handles creation of synthetic methods and vertices to gracefully invoke static code blocks.
+ *
+ * <p>Consider this example: class StaticBlockClass { static { System.debug("inside static block
+ * 1"); } static { System.debug("inside static block 2"); } } In Jorje's compilation structure,
+ * static blocks are represented like this: class StaticBlockClass { private static void <clint>() {
+ * { System.debug("inside static block 1"); } { System.debug("inside static block 2"); } } }
+ *
+ * <p>Having multiple block statements inside a method breaks SFGE's makes handling code blocks in
```

Comment:
This sentence seems like it's missing a bit?

Reply:
Looks like google formatter mauled the documentation. Let me try fixing this.